spec/features/restaurant/user_creates_new_restaurant_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+feature 'user adds a new restaurant', %{
+  As a user
+  I want to visit the new restaurant page
+  so I can add a new restaurant
+} do
+
+  # [ ] - I can navigate to the restaurants/new page
+  # [ ] - I can create a new restaurant
+  # [ ] - I am redirected to the restaurants index page
+
+  restaurant = FactoryGirl.build(:restaurant)
```

Comment:
You shouldn't be using normal variable assignment outside of your `scenario`s. I'd recommend using a `let` statement here instead.